docs/miscellaneous.rst

```diff
@@ -334,10 +334,10 @@ Comments are of course also not permitted and used here only for explanatory pur
         "myFile.sol": {
           // Required: keccak256 hash of the source file
           "keccak256": "0x123...",
-          // Required (unless "content" is used, see below): URL to the
-          // source file, protocol is more or less arbitrary, but a Swarm
-          // URL is recommended
-          "url": "bzzr://56ab..."
+          // Required (unless "content" is used, see below): Sorted URL(s)
```

Comment:
We did not even specify the character set of the file... I think this gets messy really quick. In the end, the compiler has to implement some sorting order and since we specify the version of the compiler, it is clear which one is used. It is just important that some order is used.